pkg/constants/etc.go

```diff
@@ -10,3 +10,12 @@ const TimestampFormat = time.RFC3339
 
 // DefaultDockerfile is the name of the default dockerfile
 const DefaultDockerfile = "Dockerfile"
+
+// SourceNameWebArchive is the name of the web archive source
+const SourceNameWebArchive = "web archive"
+
+// SourceNameGit is the name of the git source
+const SourceNameGit = "git repository"
+
+// SourceNameLocal is the name of local source
+const SourceNameLocal = "local directory"
```

Comment:
Please open an issue to merge these and enable align with Docker build.

Reply:
https://github.com/Azure/acr-builder/issues/21